packages/teams-js/src/public/meeting.ts

```diff
@@ -210,6 +210,39 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface MicState {
+    id: number;
```

Comment:
How is this id being set? My understanding was that the id needed to be set by the teamsjs-sdk so that the host could use them to make sure they were handling the requests in the right order